specs/dispute-game.md

```diff
@@ -0,0 +1,73 @@
+# Dispute Game
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Attestation Dispute Game](#attestation-dispute-game)
+  - [Smart Contract Implementation](#smart-contract-implementation)
+    - [Attestation Structure](#attestation-structure)
+  - [Why EIP-712](#why-eip-712)
+  - [Offchain Actor](#offchain-actor)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Attestation Dispute Game
+
+The attestation based dispute game is meant to be a dispute game based on social consensus that
```

Comment:
Comment by @refcell 


> The attestation based dispute game shifts the current permissioned system to a permissionless,
	social-consensus based architecture that can progressively decentralize over time with larger
	participant sets. In this "game", output proposals are permissionless, hence anyone can post an
	output. To prevent invalid outputs, a social quorum can revert outputs when an invalid one is found.
	A set of attestors is maintained and [EIP-712](https://eips.ethereum.org/EIPS/eip-712) signatures
	over canonical output roots can be used as attestations.


https://github.com/ethereum-optimism/optimism/pull/5270/files#r1153451353